conformance-suites/2.0.0/conformance/extensions/webgl-compressed-texture-etc.html

```diff
@@ -138,10 +139,18 @@
     shouldBe("formats.length", isPositive ? "10" : "0");
 
     debug("");
-    shouldThrow("gl.compressedTexImage2D(gl.TEXTURE_2D, 0, COMPRESSED_R11_EAC, 4, 4, 0, null)");
-    shouldThrow("gl.compressedTexSubImage2D(gl.TEXTURE_2D, 0, 0, 0, 0, 0, COMPRESSED_R11_EAC, null)");
-    shouldThrow("gl.compressedTexImage3D(gl.TEXTURE_2D_ARRAY, 0, COMPRESSED_R11_EAC, 4, 4, 4, 0, null)");
-    shouldThrow("gl.compressedTexSubImage3D(gl.TEXTURE_2D_ARRAY, 0, 0, 0, 0, 0, 0, 0, COMPRESSED_R11_EAC, null)");
+    if (contextVersion >= 2) {
+      // `null` coerces into `0` for the PBO entrypoint, yielding INVALID_OP due to no PBO bound.
```

Comment:
We have absolutely no tests to cover the compressedTex{Sub}Image from PBOs in the conformance test suite until this case.  It turns out Chrome didn't add the idl function signatures for compressedTex{Sub}Image with PBOs, that's why we are throwing in WebGL1 and WebGL2.  At this point, the ToT change is fine, but the 2.0.0 case needs to allow both behaviors.  Let's merge this and I'll upload a pull request for that.